src/ddsrt/tests/CMakeLists.txt

```diff
@@ -10,9 +10,11 @@
 # SPDX-License-Identifier: EPL-2.0 OR BSD-3-Clause
 #
 include(CUnit)
+include(GenerateExportHeader)
 
 set(sources
     "atomics.c"
+    "dl.c"
```

Comment:
just like "library" is a bit vague, "dl" is a bit vague, too

Reply:
dynlib